app/components/autocomplete/autocomplete_section_header.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+import {ActivityIndicator, View} from 'react-native';
```

Comment:
the activityIndicator is fine for now, we should ask UX (for a new ticket) if they want to change this with the lottie loading spinner that we added to the project